draft-irtf-cfrg-frost.md

```diff
@@ -195,6 +195,7 @@ We now detail a number of member functions that can be invoked on `G`.
 
 - Order(): Outputs the order of `G` (i.e. `p`).
 - Identity(): Outputs the identity `Element` of the group (i.e. `I`).
+- RandomBytes(): Outputs random bytes to fit a Scalar element, in little-endian order.
```

Comment:
```suggestion
```

I wouldn't make this a function for the group -- it's sort of just a routine that exists.